docs/swarm/swarm-tutorial/drain-node.md

```diff
@@ -97,7 +97,7 @@ task assignments for the `redis` service:
     9bg7cezvedmkgg6c8yzvbhwsd  redis.3       redis:3.0.6  worker2   Running        Running 4 minutes
     ```
 
-    The Swarm manager maintains the desired state by ending the task on a node
+    The swarm manager maintains the desired state by ending the task on a node
```

Comment:
Either wording seems correct to me. The original wording emphasizes the low-level actions the manager takes (ending a task, creating a new task), and your alternative emphasizes the final outcome (a task is replaced by one on a different node).